src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -2008,9 +2018,12 @@ private void OnFrameRendered()
 
         private void OnScreenshotAcquired(IntPtr data)
         {
-            ImageView image = new ImageView(data, true);
-            screenshotAcquiredCallback?.Invoke(image);
-            image.Dispose();
+            if (screenshotImage != null)
+            {
+                screenshotImage.Dispose();
+            }
+            screenshotImage = new ImageView(data, true);
+            screenshotAcquiredCallback?.Invoke(screenshotImage);
```

Comment:
I thinks you should not dispose screenshotimage even though warning.
warning. A lifetime of captured image should be decided by API caller

```suggestion
            screenshotAcquiredCallback?.Invoke(new ImageView(data, true););
```

Reply:
Done.